sklearn/metrics/_ranking.py

```diff
@@ -287,9 +287,10 @@ def det_curve(
 ):
     """Compute Detection Error Tradeoff (DET) for different probability thresholds.
 
-    .. note::
-       This metric is used for evaluation of ranking and error tradeoffs of
-       a binary classification task.
+    Note: this implementation is restricted to the :term:`binary` classification task.
+
+    DET curve is used for evaluation of ranking and error tradeoffs of binary
+    classification tasks.
```

Comment:
The note box makes this entry a bit busy:

<img width="1348" height="866" alt="Image" src="https://github.com/user-attachments/assets/7f0361b5-c509-4e90-b0c6-fc12e73dcb9f" />

I've removed the box, making it just text, similar to what is in the `precision_recall_curve` and `roc_curve` API entries.